scripts/operations/node_management/ncn-image-modification.sh

```diff
@@ -532,6 +532,37 @@ EOF
   done
 }
 
+function csm_145_patch {
+  local rpms=(
+    "${CSM_PATH}/rpm/cray/csm/sle-15sp4/noarch/dracut-metal-mdsquash-2.3.2-1.noarch.rpm"
+  )
+
+  if [ ! -d "${CSM_PATH}" ]; then
+    echo >&2 "CSM_PATH does not exist, or was not defined."
+    return 1
+  fi
+
+  echo "Patching images for CSM 1.4.5 ... "
+  for squash in "${SQUASH_PATHS[@]}"; do
+    (
+      pushd "$(dirname "$squash")" || exit
+
+      if rpm -Uvh --nodeps --root "$(pwd)/squashfs-root" "${rpms[@]}"; then
+        :
+      else
+        rc=$?
+        [ "$rc" -ne 127 ] && return "$rc"
+      fi
+
+      sed -i -r -E 's|(.*)\[\s?LABEL=CONTAIN,\s+/var/lib/containers.*\]|\1[ LABEL=CONTAIN, /var/lib/containers, auto, "defaults" ]|g' \
```

Comment:
The quotes seem extraneous, given the other strings didn't. I figure these aren't needed either.

```suggestion
      sed -i -r -E 's|(.*)\[\s?LABEL=CONTAIN,\s+/var/lib/containers.*\]|\1[ LABEL=CONTAIN, /var/lib/containers, auto, defaults ]|g' \
```